src/plugins/workspace/public/components/workspace_initial/workspace_initial.tsx

```diff
@@ -3,184 +3,245 @@
+    services: { application, chrome, workspaces, http },
   } = useOpenSearchDashboards<CoreStart>();
-  const isDashboardAdmin = application.capabilities.dashboards?.isDashboardAdmin;
-  const logos = chrome.logos;
-  const createWorkspaceUrl = application.getUrlForApp(WORKSPACE_CREATE_APP_ID, { absolute: true });
-  const settingsAndSetupUrl = application.getUrlForApp('settings_landing', { absolute: true });
-  const isDarkTheme = uiSettings.get('theme:darkMode');
-  const backGroundUrl = isDarkTheme ? BackgroundDarkSVG : BackgroundLightSVG;
+  const isDashboardAdmin = !!application.capabilities.dashboards?.isDashboardAdmin;
+  const availableUseCases = registeredUseCases$
+    .getValue()
+    .filter((item) => !item.systematic || item.id === 'all');
+  const workspaceList = workspaces.workspaceList$.getValue();
+  const [isUseCaseFlyoutVisible, setIsUseCaseFlyoutVisible] = useState(false);
+  const [defaultUseCaseId, setDefaultUseCaseId] = useState(availableUseCases[0].id);
+
+  const handleClickUseCaseInformation = useCallback((useCaseId: string) => {
+    setIsUseCaseFlyoutVisible(true);
+    setDefaultUseCaseId(useCaseId);
+  }, []);
+  const handleFlyoutClose = useCallback(() => {
+    setIsUseCaseFlyoutVisible(false);
+  }, []);
+
+  const useCaseCards = availableUseCases.map((useCase) => {
+    return (
+      <EuiFlexItem key={useCase.id}>
+        <WorkspaceUseCaseCard
+          useCase={useCase}
+          workspaces={workspaceList}
+          application={application}
+          http={http}
+          isDashboardAdmin={isDashboardAdmin}
+          handleClickUseCaseInformation={handleClickUseCaseInformation}
+        />
+      </EuiFlexItem>
+    );
+  });
+
+  const mountUserAccountRef = useRef<HTMLDivElement>(null);
+  const mountSettingRef = useRef<HTMLDivElement>(null);
+  const mountDevToolsRef = useRef<HTMLDivElement>(null);
+
+  const [isCreateWorkspacePopoverOpen, setIsCreateWorkspacePopoverOpen] = useState(false);
+  const [userAccountMount, setUserAccountMount] = useState<ChromeNavControl | undefined>(undefined);
+  const [settingMount, setSettingMount] = useState<ChromeNavControl | undefined>(undefined);
+  const [devToolsMount, setDevToolsMount] = useState<ChromeNavControl | undefined>(undefined);
+
+  useEffect(() => {
+    const subscription = chrome.navControls.getLeftBottom$().subscribe((items) => {
+      setSettingMount(items.at(2));
+      setDevToolsMount(items.at(3));
+      setUserAccountMount(items.at(-1));
+    });
+
+    return () => {
+      subscription.unsubscribe();
+    };
+  }, [chrome.navControls]);
+
+  useEffect(() => {
+    if (
+      userAccountMount?.mount &&
+      settingMount?.mount &&
+      devToolsMount?.mount &&
+      mountUserAccountRef.current &&
+      mountSettingRef.current &&
+      mountDevToolsRef.current
+    ) {
+      userAccountMount.mount(mountUserAccountRef.current);
+      devToolsMount.mount(mountDevToolsRef.current);
+      settingMount.mount(mountSettingRef.current);
```

Comment:
We should mount the menu in individual `if else`. The code won't work if security plugin is not installed.

Reply:
Sure, I will update it.